system-tests/project-fixtures/svelte/src/mount.cy.js

```diff
@@ -53,4 +53,34 @@ describe("Svelte mount", () => {
     cy.then(() => messageStore.set("Written from spec"));
     cy.contains("h1", "Written from spec");
   });
+
+  context("log", () => {
+    it("displays component name in mount log", () => {
+      cy.mount(Counter);
+
+      cy.wrap(Cypress.$(window.top.document.body)).within(() =>
+        cy
+          .contains("displays component name in mount log")
+          .closest(".collapsible")
+          .click()
+          .within(() =>
+            cy
+              .get(".command-name-mount")
+              .should("contain", "mount<Counter ... />")
```

Comment:
Maybe it would be helpful for future us to add a comment detailing the fact there is no space here purposefully, but I'm not too concerned with it.